### PR TITLE
Add Minio Helm Chart

### PR DESCRIPTION
**Pull request by trispera**

### CODE REVIEW

File: charts/minio/templates/NOTES.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+===============================
+  CSC Minio Helm deployed
+===============================
+Congratulations! Your minio app is being installed and deployed.
+
+Get the application console URL by running this command:
+
+  export MINIO=$(oc get route --namespace={{ .Release.Namespace }} -o yaml | yq '.items[1].spec.host')
```
Comment (lvarin) [inline charts/minio/templates/NOTES.txt:8]:
```suggestion
  export MINIO=$(oc get route --namespace={{ .Release.Namespace }} -o yaml | yq '.items[] | select(.metadata.name == "{{ .Values.minio.clusterName }}-console") .spec.host´ -r)
```

File: charts/minio/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.16.0"
```
[the remark this answers is not in the document]
Comment (trispera) [reply charts/minio/Chart.yaml:24]: It's the default version generated by the command `helm create ...`

**Review by lvarin — CHANGES_REQUESTED** [review]
Check the requested changes

File: charts/minio/templates/NOTES.txt (hunk 1 above)
Comment (trispera) [reply charts/minio/templates/NOTES.txt]: Tested and approved

**Review by lvarin — APPROVED** [review]

**Pull request merged**